content/docs/tina-cloud/data-layer.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Data Layer
+last_edited: '2022-02-15T20:00:00.000Z'
+---
+
+{{ WarningCallout text="This is an experimental feature, and the API is subject to change. Have any thoughts? Let us know in the chat, or through one of our [community channels](/community/)!" }}
+
+Tina Cloud now supports an optional Data Layer which implements a database layer between the Tina's GraphQL api and the
+GitHub rest api. This data layer buffers requests between Tina Cloud and GitHub resulting in improved editing
+performance with TinaCMS. As this feature evolves, it will provide additional capabilities such as:
+- more sophisticated GraphQL queries
+- referential integrity
+- sorting
+- filtering
+- pagination 
+
+# Enabling the Data Layer
+
+The TinaCMS data layer is enabled by specifying the following TinaCMS command line flag: `--experimentalData`. We
```

Comment:
How do you feel about putting the flag in a code block? The way it's rendering now puts one dash on a separate line and I could imagine a user (ie me) trying to copy paste it and missing the first dash.

Reply:
actually never mind, it's there in the "start-tina" script. It'd be nice if the flag didn't break across lines like that but oh well